index.html

```diff
@@ -0,0 +1,219 @@
+                <a class="nav-link active" href="#"><i class="bi bi-twitter"></i></a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link active nav-social-text" href="#">Spanish</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link active nav-social-text" href="#">MyPage</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link active nav-social-text" href="#">Logout</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+      
+      <div class="navbar navbar-expand-lg bg-body-tertiary navbar-menu">
+        <div class="container-fluid">
+          <div class="d-flex nav-bar-about">
+            <a class="navbar-brand" href="./index.html"><img src="./images/mainLogo2.png" class="wttlogo"></a>
+            <ul class="navbar-nav nav-menu">
+              <li class="nav-item nav-item-home">
+                <a class="nav-link link-menu about-link" aria-current="page" href="./about.html">About</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link link-menu" href="#program">Program</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link link-menu" href="#join">Join</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link link-menu" href="#speakers">Players</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link link-menu" href="#">News</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link link-menu link-campaign" href="#">LIVE</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+    </nav>
+  </header>
+    
+  <main class="main">
+    <div class="intro-main">
+      <div class="intro">
+        <h2 class="greet">EXPERIENCE THE SINGAPORE SMASH IN STYLE</h2>
+        <h1 class="title">Singapore Smash 2023<br>World Table Tennis 2023</h1>
+        <p class="description">
+          The first Grand Smash of the year, where the world’s Top 50 players will battle it out across nine days of exciting main draw action and Singapore will once again crown a champion. <br>
+
+New for 2023, experience an unparalleled atmosphere and an unbeatable view on all the action – THE SUITES at SINGAPORE SMASH – your bespoke, high-end table tennis experience.<br>
+
+Enjoy first-class entertainment and adrenaline-filled performances with staff or clients, all from an immersive fully serviced corporate box located above the player walkout runway.
+        </p>
+        <p class="date">2023.10.13(WED) ~ 15(FRI)</p>
+        <p class="place">@Singapore, Singapore.</p>
+      </div>
+    </div>
```

Comment:
- [ ] As suggested [here](https://github.com/gdvl777/CapstoneSmash/pull/1#discussion_r1202894866), please make sure to implement the correct styles as shown in the design guidelines 🤓 The previous reviewer has beautifully explained with the images, please implement that 😎⛏

Reply:
Sorry, but the images were changed.